src/grafana/prometheus/docker/docker-compose.yml

```diff
@@ -35,7 +33,7 @@ services:
   # Use `docker exec -it service-discovery /bin/sh` to logging the container
   # Project https://github.com/tzolov/spring-cloud-dataflow-prometheus-service-discovery
   service-discovery:
-    image: tzolov/spring-cloud-dataflow-prometheus-service-discovery:0.0.2
+    image: springcloud/spring-cloud-dataflow-prometheus-service-discovery:0.0.2
```

Comment:
wasn't it decided not to use this due to the dynamic port issue?

Reply:
This is the only way to use the prometheus + grafana setup for SCDF local server. It expects though that the SCDF server is run on the host not via docker containers.

Reply:
right; maybe im misunderstanding something -- i thought on our last call prometheus on compose wasn't going to happen due to the host issue. we can chat offline if needed.